### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by niroshan**
Updates to point to GHAS Capabilities, and tighter wording on License terms

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # CodeQL
 
-This open source repository contains the standard CodeQL libraries and queries that power [LGTM](https://lgtm.com) and the other CodeQL products that [GitHub](https://github.com) makes available to its customers worldwide. For the queries, libraries, and extractor that power Go analysis, visit the [CodeQL for Go repository](https://github.com/github/codeql-go).
+This open source repository contains the standard CodeQL libraries and queries that power [GitHub Advanced Security](https://github.com/features/security/code) and the other Application Security products that [GitHub](https://github.com/features/security/) makes available to its customers worldwide. For the queries, libraries, and extractor that power Go analysis, visit the [CodeQL for Go repository](https://github.com/github/codeql-go).
```
Comment (felicitymay) [inline README.md:3]: Remove capitalization.

```suggestion
This open source repository contains the standard CodeQL libraries and queries that power [GitHub Advanced Security](https://github.com/features/security/code) and the other application security products that [GitHub](https://github.com/features/security/) makes available to its customers worldwide. For the queries, libraries, and extractor that power Go analysis, visit the [CodeQL for Go repository](https://github.com/github/codeql-go).
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 ## How do I learn CodeQL and run queries?
 
-There is [extensive documentation](https://help.semmle.com/QL/learn-ql/) on getting started with writing CodeQL.
-You can use the [interactive query console](https://lgtm.com/help/lgtm/using-query-console) on LGTM.com or the [CodeQL for Visual Studio Code](https://help.semmle.com/codeql/codeql-for-vscode.html) extension to try out your queries on any open source project that's currently being analyzed.
+There is [extensive documentation](https://codeql.github.com/docs/) on getting started with writing CodeQL.
+You can use the [CodeQL for Visual Studio Code](https://help.semmle.com/codeql/codeql-for-vscode.html) extension or the [interactive query console](https://lgtm.com/help/lgtm/using-query-console) on LGTM.com (Semmle Legacy product) to try out your queries on any open source project that's currently being analyzed.
```
Comment (felicitymay) [inline README.md:8]: Change the link from help.semmle.com to the CodeQL microsite.

```suggestion
You can use the [CodeQL for Visual Studio Code](https://codeql.github.com/docs/codeql-for-visual-studio-code/) extension or the [interactive query console](https://lgtm.com/help/lgtm/using-query-console) on LGTM.com (Semmle Legacy product) to try out your queries on any open source project that's currently being analyzed.
```

File: README.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 
 ## Contributing
 
 We welcome contributions to our standard library and standard checks. Do you have an idea for a new check, or how to improve an existing query? Then please go ahead and open a pull request! Before you do, though, please take the time to read our [contributing guidelines](CONTRIBUTING.md). You can also consult our [style guides](https://github.com/github/codeql/tree/main/docs) to learn how to format your code for consistency and clarity, how to write query metadata, and how to write query help documentation for your query.
 
 ## License
 
-The code in this repository is licensed under the [MIT License](LICENSE) by [GitHub](https://github.com).
+The code in this repository is licensed under the [MIT License](LICENSE) by [GitHub](https://github.com). The use of CodeQL on Open source code is licensed under specific [Terms & Conditions](https://securitylab.github.com/tools/codeql/license/) UNLESS you have a commercial license in place. If you'd like to use CodeQL with a commercial codebase, please [contact us](https://enterprise.github.com/contact) for further help.
```
Comment (felicitymay) [inline README.md:16]: Remove unnecessary capital.

```suggestion
The code in this repository is licensed under the [MIT License](LICENSE) by [GitHub](https://github.com). The use of CodeQL on open source code is licensed under specific [Terms & Conditions](https://securitylab.github.com/tools/codeql/license/) UNLESS you have a commercial license in place. If you'd like to use CodeQL with a commercial codebase, please [contact us](https://enterprise.github.com/contact) for further help.
```

**Review by felicitymay — COMMENTED** [review]
Great call to update this file. I've made suggestions for a couple of small style issues and to update one link.

File: README.md (hunk 3 above)
Comment (mischa0083) [reply README.md:16]: `https://enterprise.github.com/contact` -> `https://github.com/enterprise/contact`, it currently 301 redirects.  `enterprise.github.com` is being deprecated.

Comment (felicitymay) [reply README.md:16]: Thanks @mischa0083 

```suggestion
The code in this repository is licensed under the [MIT License](LICENSE) by [GitHub](https://github.com). The use of CodeQL on open source code is licensed under specific [Terms & Conditions](https://securitylab.github.com/tools/codeql/license/) UNLESS you have a commercial license in place. If you'd like to use CodeQL with a commercial codebase, please [contact us](https://github.com/enterprise/contact) for further help.
```

**Review by felicitymay — APPROVED** [review]
Thanks for the reminder @mischa0083 - somehow this slipped entirely off my radar 😞 

I've committed my suggested changes (with your amendment) and think this is ready to merge.

**Comment by felicitymay** [comment]
Does anyone have any suggestions for why the checks status hasn't reported back yet?

**Pull request merged**